### PR TITLE
Make websockets available in devcontainer

### PR DESCRIPTION
**Pull request by oweitman**
**Description:**  
Without this change the devcontainer cant work, because websocket requests dont target /socket.io/ location.

### CODE REVIEW

**Comment by AlCalzone** [comment]
@UncleSamSwiss I think this is one for you

**Review by UncleSamSwiss — CHANGES_REQUESTED** [review]
Code: LGTM. As I'm no longer using DevContainer and since I have no time testing this, I can't vouch for this change, but I assume @oweitman tested this thoroughly with his adapters.

PR: please add the necessary files (e.g. migration guide for existing adapters, change log, ...)

This is a "breaking change" for parcel-based adapters, so the migration guide must reference the parcel --> esbuild migration guide.

**Comment by oweitman** [comment]
1) i tested the changes directly on an running test environment.
2) another change i did not include in the PR was: i must remove the following command from postCreateCommand, because it fails, but i dont know why. i dont care if the dicovery adapter is in the dev environment, or i can remove by my self
`iob del discovery &&` 
3) i am able do change and debug the adapter in vs code
4) i dont have tested dev in the admin/react section
5) i tried to test the complete lifecycle with starting the create-adapter script from my fork with
npx oweitman/create-adapter
it downloads, but didnt start without an error message

File: templates/_devcontainer/docker-compose.yml.ts [hunk 1]
```diff
@@ -28,29 +27,14 @@ services:
...
-        build: ./parcel
-        expose:
-            - 1234
-        ports:
-            - '1235:1235'
-        volumes:
-            - ..:/workspace:cached
-        environment:
-            - CHOKIDAR_USEPOLLING=1
-`) : ""}
```
Comment (AlCalzone) [inline templates/_devcontainer/docker-compose.yml.ts:43]: Why is all this being removed? Same for the other removals

Comment (oweitman) [reply templates/_devcontainer/docker-compose.yml.ts:43]: Ahm?
you  changed the build system to esbuild and parcel isn’t called anymore
So this Container is useless
https://github.com/ioBroker/create-adapter/blob/master/docs/updates/20220215_esbuild.md
https://github.com/ioBroker/adapter-dev/commit/b6036c6bd32d2f0e713ae5c5b4f3cab60917ce63

**Pull request closed without merging**